test/polkit/meson.build

```diff
@@ -20,8 +20,8 @@ foreach test_unit: test_units
 
   test(
     test_unit,
-    exe,
-    env: test_env,
-    is_parallel: false,
+    test_wrapper,
+    args: ['--data-dir', test_data_dir, exe.full_path()],
```

Comment:
Is this applicable to still-supported F39, which we use for CI?

Reply:
Yes, thanks to the last patch the tests run in all Packit CIs, including the F39 one, and everything seems to pass there.